### DISCUSSION TITLE
How to mannually interrupt when outputing answers?

### DISCUSSION THREAD

**Discussion by NeverOccurs** [cheshire-cat-ai/core#748] [Q&A]
Hi I really love your works. Just a quick question, when I use local ollama models sometimes it produces gibberish and never stops. I have to restart the container and it is quite annoying. How to mannually interrupt a generating process when I don't want it to continue? Many thanks in advance!

**Comment by pieroit** [comment]
> Hi I really love your works. Just a quick question, when I use local ollama models sometimes it produces gibberish and never stops. I have to restart the container and it is quite annoying. How to mannually interrupt a generating process when I don't want it to continue? Many thanks in advance!

There should be a stop sequence option in Ollama you can configure directly from tha cat, let me check
